aclk/README.md

```diff
@@ -1 +1,8 @@
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+This is the agent cloud link (ACLK) information file.
+
+The installer does not yet have steps to pull and build the local library.
+cd project_root
+git clone ssh://git@github.com/netdata/mosquitto mosquitto/
+(cd mosquitto/lib && make)      # Ignore the cpp error
```

Comment:
please use

```
text
```

see current

> https://github.com/netdata/netdata/blob/underhood-patch-1/aclk/README.md

doesnt look good

Reply:
Hi this is only temporary for internal use. Not user facing documentation.

Reply:
> internal use

if internal use implies reading... 😄